cmd/epp/main.go

```diff
@@ -171,6 +190,10 @@ func run() error {
 	datastore := datastore.NewDatastore(ctx, pmf)
 
 	scheduler := scheduling.NewScheduler(datastore)
+	if schedulerV2 == "true" {
+		setupLog.Info("Creating scheduler with prefixCache plugin", "prefix cache config", prefixCacheConfig)
+		scheduler = scheduling.NewSchedulerV2(datastore, prefixCacheConfig)
```

Comment:
I know it’s WIP, but you may consider using func NewSchedulerWithConfig to reduce the noise.
(no need for NewSchedulerv2)